### DISCUSSION TITLE
Blank page and very slow startup

### DISCUSSION THREAD

**Discussion by pitav** [nuxt/nuxt#20542] [Questions]
I built a nuxt 3 and firestore app that uses firebase hosting.

For Firestore, I use the VueFire module only on the client side through a `firebase.client.ts` file. Everything works very well... except the start of the application which starts with a blank page and can take more than 10 seconds! (the page reloads normally)

There are two parts in my application. The site part open to all and the application part accessible when you are connected.

My `nuxt.config` file:

```
export default defineNuxtConfig({
  typescript: { shim: false },

  modules: [
    'nuxt-vuefire',
    '@nuxtjs/google-fonts',
    '@nuxtjs/tailwindcss',
    'nuxt-icon',
    'nuxt-headlessui',
    '@formkit/nuxt',
    'nuxt-lodash',
    '@vueuse/nuxt',
    [
      '@pinia/nuxt',
      {
        autoImports: ['defineStore', 'storeToRefs', 'acceptHMRUpdate'],
      },
    ],
  ],
  build: {
    transpile: ['@vuepic/vue-datepicker'],
  },
  headlessui: {
    prefix: 'Headless',
  },
  vuefire: {
    auth: true,
    config: {
      apiKey: '*****',
      authDomain: '*****',
      projectId: '*****',
      storageBucket: '*****',
      messagingSenderId: '*****',
      appId: '*****',
      measurementId: '*****',
    },
  },
  googleFonts: {
    families: {
      Roboto: {
        wght: [300, 400, 500, 700, 900],
      },
      'Roboto Condensed': {
        wght: [400, 500],
      },
    },
    download: false,
    useStylesheet: true,
  },
  css: ['~/node_modules/lite-youtube-embed/src/lite-yt-embed.css'],
  imports: {
    dirs: ['stores'],
  },
  routeRules: {
    '/formation/**': { ssr: false },
    '/admin/**': { ssr: false },
  },
  nitro: {
    preset: 'firebase',
    replace: {
      [`functions.https.onRequest`]: `functions.region('europe-west3').https.onRequest`,
    },
  },
})
```
My `app.vue` file
```
<template>
  <NuxtLayout>
    <NuxtPage />
  </NuxtLayout>
</template>
```

I have absolutely no idea how to get the application to start without these 10 second waits with a blank page.

Thank you for your help

**Comment by warflash** [comment]
Sounds like you might be running into cold starts? Hard to tell without seeing metrics/logs etc.

**Reply by pitav** [reply]
here is my development url: https://form-en-plus-v2.web.app/
Thanks for your help

**Reply by warflash** [reply]
Yeah so seems like it's only happening after a few minutes of no requests coming in which makes it highly look like cold starts
